applications/mne_scan/plugins/rtfwd/rtfwd.cpp

```diff
@@ -163,6 +163,8 @@ void RtFwd::unload()
 
 bool RtFwd::start()
 {
+    // Maybe we can move all of this to the run() method?
```

Comment:
The idea on having that in start() was to check if all minimum necessary filepaths are set. You can't set them during run() so you should check before the measurement is started.